test/e2e/examples_test.go

```diff
@@ -5,45 +5,85 @@ package e2e
 import (
 	"testing"
 
-	"github.com/stretchr/testify/suite"
+	"k8s.io/client-go/kubernetes"
 
+	"github.com/argoproj/argo-workflows/v3/config"
+	"github.com/argoproj/argo-workflows/v3/pkg/client/clientset/versioned"
 	"github.com/argoproj/argo-workflows/v3/test/e2e/fixtures"
 	fileutil "github.com/argoproj/argo-workflows/v3/util/file"
+	"github.com/argoproj/argo-workflows/v3/util/kubeconfig"
 	"github.com/argoproj/argo-workflows/v3/util/logging"
+	"github.com/argoproj/argo-workflows/v3/util/secrets"
 	"github.com/argoproj/argo-workflows/v3/workflow/common"
+	"github.com/argoproj/argo-workflows/v3/workflow/hydrator"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type ExamplesSuite struct {
-	fixtures.E2ESuite
-}
+func TestExampleWorkflows(t *testing.T) {
+	restConfig, err := kubeconfig.DefaultRestConfig()
+	if err != nil {
+		t.Fatal(err)
+	}
+	kubeClient, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		t.Fatal(err)
+	}
+	configController := config.NewController(fixtures.Namespace, common.ConfigMapName, kubeClient)
+	ctx := logging.TestContext(t.Context())
+	config, err := configController.Get(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	persistence := fixtures.NewPersistence(ctx, kubeClient, config)
 
-func (s *ExamplesSuite) BeforeTest(suiteName, testName string) {
-	s.E2ESuite.BeforeTest(suiteName, testName)
-	s.Given().KubectlApply("../../examples/configmaps/simple-parameters-configmap.yaml", fixtures.NoError)
-}
+	clientset, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		t.Fatal(err)
+	}
+	sec, err := clientset.CoreV1().Secrets(fixtures.Namespace).Get(ctx, secrets.TokenName("argo-server"), metav1.GetOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
 
-func (s *ExamplesSuite) TestExampleWorkflows() {
-	ctx := logging.TestContext(s.T().Context())
-	err := fileutil.WalkManifests(ctx, "../../examples", func(path string, data []byte) error {
+	err = fileutil.WalkManifests(ctx, "../../examples", func(path string, data []byte) error {
 		wfs, err := common.SplitWorkflowYAMLFile(ctx, data, true)
 		if err != nil {
-			s.T().Fatalf("Error parsing %s: %v", path, err)
+			t.Fatalf("Error parsing %s: %v", path, err)
 		}
 		for _, wf := range wfs {
-			if _, ok := wf.GetLabels()["workflows.argoproj.io/test"]; ok {
-				s.T().Logf("Found example workflow at %s with test label\n", path)
-				s.Given().
+			_, noTestBrokenLabelExists := wf.GetLabels()["workflows.argoproj.io/no-test-broken"]
+			_, noTestBrokenEnvironmentLabelExists := wf.GetLabels()["workflows.argoproj.io/no-test-environment"]
```

Comment:
I think we should just use the previous test label, or in the very least just invert the logic by using "no-test". I don't see a point in introducing two new labels to avoid testing a workflow.